patches/third_party-rust-serde_json_lenient-v0_2-wrapper-lib.rs.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs b/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs
+index 0cfb702eb8a13be14fb1b845a1f297eb2f7b4859..ad381fd275391e9fac27c6a90c14efffd28255c6 100644
+--- a/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs
++++ b/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs
+@@ -54,6 +54,10 @@ mod ffi {
+             ctx: Pin<&'a mut ContextPointer>,
+             key: &'f str,
+         ) -> Pin<&'a mut ContextPointer>;
++        fn list_append_i64(self: &Functions, ctx: Pin<&mut ContextPointer>, val: i64);
```

Comment:
I wonder if there's a better way to add this functionality by adding new code in brave and then overriding the chromium cpp code to call that. I think at least these new functions could be implemented somewhere else to reduce the patching here for lib.rs and wrapper/functions.h?

Reply:
I'm obviously in favour of that, because this is being used in conjunction with serde to determine which function to call. I've tried to think a way to approach this, but I cant come up with a better way. We could potentially drop the list_append variants though. It depends if we have to support it in lists of values.

Reply:
cc @antonok-edm